README.md

```diff
@@ -1001,6 +1001,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [anvil.works](https://anvil.works) - Web app development with nothing but Python. Free tier with unlimited apps, 30 second timeouts.
   * [configure.it](https://www.configure.it/) — Mobile app development platform, free for 2 projects, limited features but no resource limits
   * [codenameone.com](https://www.codenameone.com/) — Open source, cross platform, mobile app development toolchain for Java/Kotlin developers. Free for commercial use with unlimited number of projects
+  * [Cosmonic](https://cosmonic.com) - Feature rich WebAssembly PaaS and SDKs for low boilerplate, flexible, and secure by default microservices. Always free tier includes a managed host, 25 microservices, and included capabilities like a robust keyvalue store, load balanced public HTTP endpoints, and more.
```

Comment:
Can you link to a url that explains the always free tier?

Reply:
Hey @ripienaar , sure thing. From the initial developer preview announcement, https://cosmonic.com/docs/releases/2022-09-29-developer-preview details it.